src/main/java/com/epam/lowcost/controller/UserController.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.lowcost.controller;
+
+import com.epam.lowcost.model.User;
+import com.epam.lowcost.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+@Controller
+@RequestMapping(value = "/user")
+
+public class UserController {
+
+    @Autowired
+    UserService userService;
+
+    @GetMapping(value = "/all")
+    public String listAllUsers(Model model) {
+        model.addAttribute("users", userService.getAllUsers());
+        return "users";
+    }
+
+    @PostMapping(value = "/id")
+    public String getById(@RequestParam long id, Model model) {
+        model.addAttribute("user", userService.getById(id));
+        return "users";
+    }
+
+    @PostMapping(value = "/add")
+    public String addUser(@RequestParam Map<String, String> params, Model model) {
+        model.addAttribute("user", userService.addUser(new User(1,
```

Comment:
It is not a readable, you have many inner constructions. Maybe split these ?

Reply:
Changed to User.builder usage.